### PR TITLE
Update on course project

### PR DESCRIPTION
**Pull request by kgalushchak**
What was done:
* implemented locally stored food images
* fixed a bug with snake disappearing after eating the food on the edge + some optimisation

### CODE REVIEW

File: course-project/src/images/energy-drink.svg [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	c-20.193,5.521-44.922,9.278-71.515,10.866C281.067,454.424,277.873,458.022,278.124,462.211z"/>
+<path d="M345.606,128.1l-8.593,2.76c-25.87,8.308-52.749,11.861-79.89,10.561c-3.238-0.156-6.206,1.754-7.412,4.754l-56.221,139.792
+	c-0.941,2.341-0.658,4.997,0.754,7.089c1.413,2.091,3.772,3.344,6.295,3.344h44.73l-42.938,156.009
+	c-1.008,3.662,0.844,7.506,4.335,9.001c0.97,0.415,1.986,0.614,2.986,0.614c2.599,0,5.108-1.338,6.519-3.689l118.519-197.532
+	c1.408-2.347,1.446-5.271,0.096-7.652c-1.348-2.382-3.874-3.855-6.611-3.855h-27.35l53.933-110.631
+	c1.333-2.736,0.906-6.004-1.087-8.305C351.679,128.058,348.506,127.169,345.606,128.1z M281.841,253.562
+	c-1.148,2.354-1.002,5.135,0.387,7.357c1.389,2.222,3.823,3.571,6.443,3.571h26.087l-83.033,138.388l30.842-112.061
+	c0.629-2.286,0.155-4.735-1.281-6.621c-1.438-1.886-3.672-2.993-6.044-2.993h-43.456l50.033-124.406
+	c24.249,0.619,48.259-2.392,71.571-8.975L281.841,253.562z"/>
+<g>
```
Comment (avilde) [inline course-project/src/images/energy-drink.svg]: This certainly doesn't look right.

I suggest running all your SVG files through this tool to optimise them and remove unnecessary props/nodes.

https://www.svgviewer.dev/

File: course-project/src/images/pill.svg [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	c45.327,45.327,118.818,45.327,164.146,0c4.567-4.567,8.548-9.483,12.195-14.565C448.045,493.23,389.378,489.366,351.638,451.627z"
+	/>
+<path style="fill:#EF111A;" d="M338.129,173.944L173.968,338.106L33.999,198.137c-45.333-45.333-45.333-118.768,0-164.101
+	c1.405-1.405,2.871-2.75,4.337-4.094c0.978-0.856,1.894-1.649,2.871-2.505C62.775,9.171,89.413,0.008,116.05,0.008
+	c29.692,0,59.445,11.302,82.111,33.969L338.129,173.944z"/>
+<path style="fill:#CA0000;" d="M60.393,160.381l145.649,145.649l-32.074,32.074L33.999,198.137
+	c-45.333-45.333-45.333-118.768,0-164.101c1.405-1.405,2.871-2.75,4.337-4.094c0.978-0.856,1.894-1.649,2.871-2.505
+	c2.444-1.955,4.888-3.789,7.332-5.62C18.787,64.035,22.635,122.624,60.393,160.381z"/>
+<ellipse style="fill:#FF6D58;" cx="113.337" cy="41.455" rx="34.564" ry="18.36"/>
+<g>
```
Comment (avilde) [inline course-project/src/images/pill.svg]: As usual, exports from design tools are badly formatted and with extra clutter (comment at top).

Use online tools like this one to greatly improve SVG file readability, size: https://www.svgviewer.dev/

File: course-project/src/images/energy-drink.svg (hunk 1 above)
Comment (kgalushchak) [reply course-project/src/images/energy-drink.svg]: Optimized images

File: course-project/src/images/pill.svg (hunk 2 above)
Comment (kgalushchak) [reply course-project/src/images/pill.svg]: Optimized images

**Pull request merged**